astro_2.0/features/DaoDashboardHeader/components/DaoLetterHeadSection/DaoLetterHeadSection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+import cn from 'classnames';
+import { MainLayout } from 'astro_3.0/features/MainLayout';
+import { DaoLogo } from 'astro_2.0/features/DaoDashboardHeader/components/DaoLogo';
+import { CopyButton } from 'astro_2.0/components/CopyButton';
+import { useAppVersion } from 'hooks/useAppVersion';
+import { DAO } from 'types/dao';
+
+import styles from 'astro_2.0/features/DaoDashboardHeader/DaoDashboardHeader.module.scss';
+import { DaoSocialLinks } from 'astro_2.0/features/DaoDashboardHeader/components/DaoSocialLinks';
+import { ProposalVariant } from 'types/proposal';
+import { IconButton } from 'components/button/IconButton';
+
+interface Props {
+  dao: DAO;
+  onCreateProposal: (
+    variant: ProposalVariant,
+    initialValues: Record<string, unknown>
+  ) => void;
+}
+
+export const DaoLetterHeadSection: FC<Props> = ({ dao, onCreateProposal }) => {
+  const { flagCover, flagLogo, daoVersion, links } = dao;
+  const { appVersion } = useAppVersion();
+
+  function getBackgroundImage() {
+    if (appVersion === 3) {
```

Comment:
`appVersion === 3` is being used pretty often in the file. Can we move it to a variable and reuse it?